packages/docusaurus-theme-classic/src/theme/Toggle/styles.module.css

```diff
@@ -103,6 +103,3 @@
   justify-content: center;
   width: 10px;
 }
-.toggle::before {
-  position: absolute;
-}
```

Comment:
Is this legacy code? Don't understand its effect before...

Reply:
Yes, it's a meaningless code.